app/src/main/java/chat/rocket/android/wallet/WalletDBInterface.kt

```diff
@@ -106,6 +108,48 @@ class WalletDBInterface {
         }
     }
 
+    fun updateTransactions(senderId:String, recipientId:String, hash:String){
+
+        thread(true){
+            var senderTransactionRecords = dynamoDBMapper?.load(TransactionRecordsDO::class.java, senderId)
+            var recipientTransactionRecords = dynamoDBMapper?.load(TransactionRecordsDO::class.java, recipientId)
+
+            if (senderTransactionRecords == null){
+                val transactionRecord = TransactionRecordsDO()
+                transactionRecord.account = senderId
+                transactionRecord.transactions = emptyList()
+                dynamoDBMapper?.save(transactionRecord)
+            }
+            if (recipientTransactionRecords == null){
+                val transactionRecord = TransactionRecordsDO()
+                transactionRecord.account = recipientId
+                transactionRecord.transactions = emptyList()
+                dynamoDBMapper?.save(transactionRecord)
+            }
+
+            senderTransactionRecords = dynamoDBMapper?.load(TransactionRecordsDO::class.java, senderId)
```

Comment:
maybe put these 2 lines in the if null blocks above so we don't load the same thing twice in the not null case